### PR TITLE
V1.4: Enhancement to approve/reject based on priority level

### PR DESCRIPTION
**Pull request by Hafizuddin-github**
- Admins can now approve each other leave request.
- Bug fixed after user priority level changed
- Added sample leave applications

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/DeleteLeaveCommandTest.java [hunk 1]
```diff
@@ -64,8 +65,7 @@ public void execute_invalidIndexUnfilteredList_throwsCommandException() {
 
     @Test
     public void execute_validIndexFilteredList_success() {
-        showLeaveAtIndex(model, INDEX_FIRST_PERSON);
-
+        //showLeaveAtIndex(model, INDEX_FIRST_PERSON);
```
Comment (Woonhian) [inline src/test/java/seedu/address/logic/commands/DeleteLeaveCommandTest.java]: The comment can be removed if not using in the future

File: src/main/java/seedu/address/logic/commands/EditLeaveCommand.java [hunk 2]
```diff
@@ -39,7 +38,9 @@
     public static final String MESSAGE_ALREADY_APPROVE = "This leave application is already approved.";
     public static final String MESSAGE_ALREADY_REJECTED = "This leave application is already rejected.";
     public static final String MESSAGE_INVALID_LEAVE_APPROVAL =
-            "Not authorized to approve leave application.";
+            "Not authorized to approve/reject this leave application.";
+    public static final String MESSAGE_INVALID_LEAVE_APPROVAL2 =
+            "Not authorized to approve/reject this leave application only another ADMIN can approve your leave.";
```
Comment (Woonhian) [inline src/main/java/seedu/address/logic/commands/EditLeaveCommand.java]: The string for this can be made a little clearer.

**Review by Woonhian — COMMENTED** [review]
Good job in updating the testcases and fixing of bugs. For this PR, maybe you could add some leaves data so that we can use it for the next demo.

File: src/main/java/seedu/address/logic/commands/EditLeaveCommand.java (hunk 2 above)
Comment (Hafizuddin-github) [reply src/main/java/seedu/address/logic/commands/EditLeaveCommand.java]: What do you mean clearer?

Comment (Woonhian) [reply src/main/java/seedu/address/logic/commands/EditLeaveCommand.java]: What kind of admin do you mean? IT Admin? Administrator? I feel that the sentence is like a little vague.

**Review by Woonhian — APPROVED** [review]
Good job in adding additional sample leave to make the product look more realistic. Coveralls also looks good, PR approved!

**Pull request merged**